pkg/scripts_test/consts_windows.go

```diff
@@ -23,10 +24,10 @@ const (
 	installTokenEnv string = "SUMOLOGIC_INSTALLATION_TOKEN"
 	apiBaseURL      string = "https://open-collectors.sumologic.com"
 
-	commonConfigPathFilePermissions uint32 = 0550
-	configPathDirPermissions        uint32 = 0550
-	configPathFilePermissions       uint32 = 0440
-	confDPathFilePermissions        uint32 = 0644
-	etcPathPermissions              uint32 = 0551
-	opampDPermissions               uint32 = 0750
+	commonConfigPathFilePermissions uint32 = 0777
+	configPathDirPermissions        uint32 = 0777
+	configPathFilePermissions       uint32 = 0666
+	confDPathFilePermissions        uint32 = 0777
+	etcPathPermissions              uint32 = 0777
+	opampDPermissions               uint32 = 0777
```

Comment:
While I'm sure there is a good reason for it, this appears to be quite liberal permissions, maybe you could add a comment explaining why?

Reply:
Will do. To be honest, I should really use Windows ACLs here and make more detailed assertions. Using os.Stat like we do for Windows or Unix, we can only really get 0444 for readonly files and 0666 for writable ones (+0111 for directories). So currently all this checks is whether the files are writable by their owner (which is the LocalSystem user who owns files for shared Programs.)

Reply:
I added a comment explaining these. Looks like Windows ACLs aren't trivial to check, so I'm going to merge this PR, and add those in a followup.